applications/EmpireApplication/python_scripts/co_simulation_solvers/co_simulation_base_coupling_solver.py

```diff
@@ -161,3 +161,6 @@ def Check(self):
 
     def IsDistributed(self):
         return True
+
+    def _Name(self):
```

Comment:
Same, do you need this in your workflow?

Reply:
removed, and works fine